kolibri/plugins/facility/assets/src/modules/classAssignMembers/handlers.js

```diff
@@ -34,7 +33,7 @@ export function showLearnerClassEnrollmentPage(store, toRoute) {
     ([facilityUsers, classroom, classUsers]) => {
       store.commit('classAssignMembers/SET_STATE', {
         facilityUsers: facilityUsers.results.map(_userState),
-        classUsers: classUsers.results.map(_userState),
+        classUsers: classUsers.results,
```

Comment:
Without pagination parameters passed to the fetchCollection, I think `classUsers` will just be the results directly, rather than having a results attribute?

Also, why are we removing the userState mapping?

Reply:
I don't really know why... originally I had both the pagination parameters and the userState, to follow the same pattern, and I kept getting a 404 error that said ID: NOT FOUND. Removing both was the only way that the results were the correct array.

Reply:
Soo..... the reason this doesn't matter is that we are already excluding non-class members from the original fetch, so there is no reason to even do this fetch any more, it's dead code.